### PR TITLE
Add Falco as an alternative to FastQC

### LINKED ISSUE #947: Add FALCO as alternative to FASTQC
Because Java sucks (sometimes)

### PR DESCRIPTION
**Pull request by scarlhoff**
closes #947

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/eager/tree/master/.github/CONTRIBUTING.md)
    - [x] If necessary, also make a PR on the nf-core/eager _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by scarlhoff** [comment]
@nf-core-bot fix linting

File: docs/output.md [hunk 1]
```diff
@@ -71,6 +71,23 @@ Results generated by MultiQC collate pipeline QC from supported tools e.g. FastQ
...
+  - `falco_raw/`
+    - `*_falco.{html,txt}`: Falco report containing quality metrics of input FASTQs.
+  - `falco_processed/`
+    - `*_falco.{html,txt}`: Falco report containing quality metrics of pipeline preprocessed FASTQs.
+
+</details>
+
+[Falco](https://doi.org/10.12688%2Ff1000research.21142.2) is an emulation of the popular FastQC software to check large sequencing reads for common problems. For further reading and documentation see the [Falco GitHub repository](https://github.com/smithlabcode/falco).
+
+Falco results in the `raw/` directory contain the results of running Falco on the 'raw' input files (with no processing by the pipeline). The `processed/` directory is for reads run through either fastp or AdapterRemoval preprocessing, and therefore will only be present if preprocessing is executed. These Falco results occur in both .html (the report) and .txt format (raw data).
```
Comment (jfy133) [inline docs/output.md:89]:
```suggestion
Falco results in the `raw/` directory contain the results of running Falco on the 'raw' input files (with no processing by the pipeline). The `processed/` directory is for reads run through either fastp or AdapterRemoval preprocessing, and therefore will only be present if preprocessing is executed. These Falco results occur in both .html (the report) and .txt format (raw data), and should provide the  same information as with FastQC.
```

File: nextflow_schema.json [hunk 2]
```diff
@@ -494,7 +502,7 @@
...
                     "description": "Specify which tool to use for deduplication.",
-                    "help_text": "Sets the duplicate read removal tool. By default uses `markduplicates` from Picard. Alternatively an ancient DNA specific read deduplication tool `dedup` (Peltzer et al. 2016) is offered. The latter utilises both ends of paired-end data to remove duplicates (i.e. true exact duplicates, as markduplicates will over-zealously deduplicate anything with the same starting position even if the ends are different).\n\n> ⚠️ DeDup can only be used on collapsed (i.e. merged) reads from paired-end sequencing.",
+                    "help_text": "Sets the duplicate read removal tool. By default uses `markduplicates` from Picard. Alternatively an ancient DNA specific read deduplication tool `dedup` (Peltzer et al. 2016) is offered. The latter utilises both ends of paired-end data to remove duplicates (i.e. true exact duplicates, as markduplicates will over-zealously deduplicate anything with the same starting position even if the ends are different).\n\n> \u26a0\ufe0f DeDup can only be used on collapsed (i.e. merged) reads from paired-end sequencing.",
```
Comment (jfy133) [inline nextflow_schema.json:505]:
```suggestion
                    "help_text": "Sets the duplicate read removal tool. By default uses `markduplicates` from Picard. Alternatively an ancient DNA specific read deduplication tool `dedup` (Peltzer et al. 2016) is offered. The latter utilises both ends of paired-end data to remove duplicates (i.e. true exact duplicates, as markduplicates will over-zealously deduplicate anything with the same starting position even if the ends are different).\n\n> ⚠️ DeDup can only be used on collapsed (i.e. merged) reads from paired-end sequencing.",
```

**Review by jfy133 — CHANGES_REQUESTED** [review]
Very minor documentation tweaks but otherwise LGTM

**Review by jfy133 — APPROVED** [review]
Just remembered you are on excavation so I've committed my suggestions and will merge in already :)

**Comment by jfy133** [comment]
@nf-core-bot fix linting

**Pull request merged**